tests/qos/test_tunnel_qos_remap.py

```diff
@@ -436,12 +436,16 @@ def test_pfc_pause_extra_lossless_active(ptfhost, fanouthosts, rand_selected_dut
                                     pfc_frames_number=PFC_PKT_COUNT,
                                     peer_info=peer_info)
 
+        dst_ports = dualtor_meta['target_server_port']
+        if not isinstance(dualtor_meta['target_server_port'], list):
+            dst_ports = [dualtor_meta['target_server_port']]
```

Comment:
Seems dualtor_meta['target_server_port'] is always a string, not a list. Why do we need this check?
https://github.com/sonic-net/sonic-mgmt/blob/66a0b08a8d75c2353e8800ce37265e19cbdeea9a/tests/common/dualtor/dual_tor_utils.py#L1162